cmd/builder/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.23-alpine3.20
+RUN apk --update add ca-certificates
+
+ARG SERVICE_NAME=ocb
+
+RUN addgroup --gid 1001 -S ${SERVICE_NAME} && \
+    adduser -G ${SERVICE_NAME} --shell /bin/false --disabled-password \
+    -H --uid 1001 ${SERVICE_NAME} && \
+    chown ${SERVICE_NAME}:${SERVICE_NAME} /usr/local/go/bin
```

Comment:
Why is it needed to change the owner of the `/usr/local/go/bin` directory?

Reply:
I was getting error messages with permissions, but this was before I added the arguments to the adduser command and the addgroup command, so I've removed it. Thanks.